prepare_symlinks.yml

```diff
@@ -0,0 +1,19 @@
+- name: Create symlink to the runme.sh script in specific target directories
```

Comment:
Please add this script to something like "tools" directory. If I understand it correclty it's for manual execution only, right?

Reply:
I'm using this script for CI, if you will look into the makefile you will see I'm adding it there:
```
.PHONY: prepare_symlinks
prepare_symlinks:
	ansible-playbook prepare_symlinks.yml
```
this approach actually make the manual testing to require creating this `runme.sh` manually for the target

Reply:
Makes complete sense. I would just move the playbook outside of root directory.

Reply:
sure, on it

Reply:
done